core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -272,4 +273,18 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext {
       sc.stop()
     }
   }
+
+  test("calling multiple sc.stop() must not throw any exception") {
+    try {
+      sc = new SparkContext(new SparkConf().setAppName("test").setMaster("local"))
+      val cnt = sc.parallelize(1 to 4).count()
+      sc.cancelAllJobs()
+      sc.stop()
+      // call stop second time
+      sc.stop()
+    } catch {
+      case e: Exception =>
```

Comment:
This `try` should be removed since it doesn't really add anything, except hiding the exception. Let it fly from the method to fail the test.